gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -124,23 +124,41 @@ def validate(self, parsed_args, client=None):
       raise ValueError(
           'Expected a table reference (PROJECT:DATASET.TABLE) '
           'instead of {}.'.format(parsed_args.output_table))
+    if not client:
+      credentials = GoogleCredentials.get_application_default().create_scoped(
+          ['https://www.googleapis.com/auth/bigquery'])
+      client = bigquery.BigqueryV2(credentials=credentials)
+    project_id = output_table_re_match.group('project')
+    dataset_id = output_table_re_match.group('dataset')
+    table_id = output_table_re_match.group('table')
     try:
-      if not client:
-        credentials = GoogleCredentials.get_application_default().create_scoped(
-            ['https://www.googleapis.com/auth/bigquery'])
-        client = bigquery.BigqueryV2(credentials=credentials)
       client.datasets.Get(bigquery.BigqueryDatasetsGetRequest(
-          projectId=output_table_re_match.group('project'),
-          datasetId=output_table_re_match.group('dataset')))
+          projectId=project_id,
+          datasetId=dataset_id))
     except exceptions.HttpError as e:
       if e.status_code == 404:
         raise ValueError('Dataset %s:%s does not exist.' %
-                         (output_table_re_match.group('project'),
-                          output_table_re_match.group('dataset')))
+                         (project_id, dataset_id))
       else:
         # For the rest of the errors, use BigQuery error message.
         raise
-
+    # Ensuring given output table doesn't already exist to avoid overwriting it.
+    args_dict = vars(parsed_args)
+    if not args_dict['append']:
```

Comment:
you can just do `if not parsed_args.append:`

Reply:
done.